Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/exceptions/AsyncExceptionHandler.java

```
@@ -0,0 +1,17 @@
+package edu.asu.conceptpower.app.exceptions;
+
+import java.lang.reflect.Method;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.aop.interceptor.AsyncUncaughtExceptionHandler;
+
+public class AsyncExceptionHandler implements AsyncUncaughtExceptionHandler {
+
+    private static final Logger logger = LoggerFactory.getLogger(AsyncExceptionHandler.class);
+
+    @Override
+    public void handleUncaughtException(Throwable ex, Method method, Object... params) {
+        logger.error("Exception while executing the async call in " + method.getName());
```

don't swallow the exception!